database.py

```diff
@@ -0,0 +1,226 @@
+from sqlalchemy import create_engine, Column, Integer, String, Table, func, inspect
+from sqlalchemy import MetaData
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.orm import mapper
+
+metadata = MetaData()
+dbname = "./Database/data.db"
+engine = create_engine(f'sqlite:///{dbname}')
+
+
+class User(object):
+    id = None
+    name = None
+
+    def __init__(self, name, id):
+        self.name = name
+        self.id = id
+
+
+class KPOP(User):
+    pass
+
+
+class BTS(User):
+    pass
+
+
+class GOT7(User):
+    pass
+
+
+class MonstaX(User):
+    pass
+
+
+class Twice(User):
+    pass
+
+
+class Blackpink(User):
+    pass
+
+
+class ROCK(User):
+    pass
+
+
+class TheBeatles(User):
+    pass
+
+
+class PinkF(User):
+    pass
+
+
+class GreenD(User):
+    pass
+
+
+class Aerosmith(User):
+    pass
+
+
+class LinkinP(User):
+    pass
+
+
+class TaylorSwift(User):
+    pass
+
+
+class BensonBoone(User):
+    pass
+
+
+class EdSheeran(User):
```

Comment:
EdSharen is not a class.... 

Do you understand difference between class and object?